docs/usage_tips/custom_field.md

```
@@ -89,6 +89,12 @@ You can manually assign a value to the `librenms_id` custom field for a device u
       ```
       For multiple servers: `{"production": 42, "staging": 17}`
 
+      An object with multiple usable mappings can also store its preferred server:
+      ```json
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)